apps/src/code-studio/dancePartySongArtistTags.js

```diff
@@ -7,7 +7,7 @@
 
 export const SongTitlesToArtistTwitterHandle = {
   backtoyou_selenagomez: 'SelenaGomez',
-  bornthisway_ladygaga: 'LadyGaga',
+  bornthisway_ladygaga: 'ladygaga',
```

Comment:
intentional to downcase this?

Reply:
yeah, i double-checked all of the artists' handles and noticed that hers is all lowercase. handles are case-insensitive in twitter, so there's no functional change, i just wanted them to match exactly 😄